### PR TITLE
added mp3 instructions

### PR DESCRIPTION
**Pull request by prithviramanathan**
Make sure to add the mp3 package to the mps folder and change the name to mp3.unitypackage

### CODE REVIEW

File: _assignments/mp3.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    points: 40
+    description: Toggling rotation tracking works correctly, regardless of where we are looking.
+  -
+    name: Depth Perception Trick
+    points: 20
+    description: |
+      The spheres appear to be of equal size while the script is active,
+      regardless of where the user is in the room.
+  -
+    name: Depth Perception Script
```
Comment (andrewipark) [inline _assignments/mp3.md]: This is easy enough to be 15

File: _assignments/mp3.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+## Disabling Tracking
+
+The Oculus software automatically sets the camera's `Transform` based on the headset's position and rotation tracking.
+In this task, you must counteract that tracking, i.e. the net effect is as though tracking was disabled.
+This should help you understand how much tracking contributes towards a feeling of presence in VR.
+
+Create a script named `ToggleTracking` that turns tracking on and off.
+You must add controls for two separate buttons to disable position and rotation tracking, independently of each other.
+Note that simultaneously disabling rotation and position tracking counts as extra credit.
+If you attempt this, make a note in your README file.
```
Comment (andrewipark) [inline _assignments/mp3.md]:
```suggestion
```
We don't offer this anymore.

File: _assignments/mp3.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+index: 30
+due: "Mar. 13, 2019, @ 11:59 PM"
+material:
+  -
+    name: MP3 Unity Package
+    url: /mps/MP3.unitypackage
+rubric:
+  -
+    name: Position Tracking
+    points: 20
```
Comment (andrewipark) [inline _assignments/mp3.md]: Recommend to change the breakdown from 20 for position and 40 for rotation, to
15 for position, 25 for rotation, 25 for both position and rotation simultaneously. note the extra 5 that we have b/c of below

File: _assignments/mp3.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+    description: |
+      The spheres appear to be of equal size while the script is active,
+      regardless of where the user is in the room.
+  -
+    name: Depth Perception Script
+    points: 20
+    description: The spheres appear in the correct order, and the script works when run repeatedly.
+points: 100
+---
+# Overview
```
[the remark this answers is not in the document]
Comment (prithviramanathan) [reply _assignments/mp3.md]: done

File: _assignments/mp3.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+This assignment will demonstrate the difference between position and rotation tracking. It will also teach you to appreciate how much depth perception relies on both of these, and teach you how to enable and disable them. It'll also help you understand the rigid body transformations covered in class.
+
+Note, make sure to have Oculus Integration downloaded/imported on your project and 'Virtual Reality Support' enabled.
```
[the remark this answers is not in the document]
Comment (prithviramanathan) [reply _assignments/mp3.md]: done

File: _assignments/mp3.md (hunk 2 above)
Comment (prithviramanathan) [reply _assignments/mp3.md]: done

**Pull request merged**